solidity/contracts/system/TBTCSystem.sol

```diff
@@ -169,30 +169,49 @@ contract TBTCSystem is Ownable, ITBTCSystem, DepositLog {
     function beginSignerFeeDivisorUpdate(uint16 _signerFeeDivisor)
         external onlyOwner
     {
-        require(_signerFeeDivisor > 9, "Signer fee divisor must be greater than 9, for a signer fee that is <= 10%.");
+        require(
+            _signerFeeDivisor > 9,
+            "Signer fee divisor must be greater than 9, for a signer fee that is <= 10%."
+        );
+        require(
+            _signerFeeDivisor < 2000,
+            "Signer fee divisor must be less than 2000, for a signer fee that is > 5bps."
```

Comment:
Possibly `> 0.005%` instead of quoting bps? We use % fairly consistently.

Reply:
That’s fair, yeah.